parser.y

```diff
@@ -471,62 +482,58 @@ Term {
 }
 
 Import:
-"import" String "as" '$' IDENT ';' {
+ImportWhat ';' {
```

Comment:
I suppose the rules for import syntax could have used less violence, but it's fine.